terraform-provider-jans/jans/api_app_configuration.go

```diff
@@ -28,43 +28,47 @@ type AssetDirMapping struct {
-	AuthIssuerUrl               string                    `schema:"auth_issuer_url" json:"authIssuerUrl"`
-	AuthOpenidConfigurationUrl  string                    `schema:"auth_openid_configuration_url" json:"authOpenidConfigurationUrl"`
-	AuthOpenidIntrospectionUrl  string                    `schema:"auth_openid_introspection_url" json:"authOpenidIntrospectionUrl"`
-	AuthOpenidTokenUrl          string                    `schema:"auth_openid_token_url" json:"authOpenidTokenUrl"`
-	AuthOpenidRevokeUrl         string                    `schema:"auth_openid_revoke_url" json:"authOpenidRevokeUrl"`
-	SmallryeHealthRootPath      string                    `schema:"smallrye_health_root_path" json:"smallryeHealthRootPath"`
-	ExclusiveAuthScopes         []string                  `schema:"exclusive_auth_scopes" json:"exclusiveAuthScopes"`
-	CorsConfigurationFilters    []CorsConfigurationFilter `schema:"cors_configuration_filters" json:"corsConfigurationFilters"`
-	LoggingLevel                string                    `schema:"logging_level" json:"loggingLevel"`
-	LoggingLayout               string                    `schema:"logging_layout" json:"loggingLayout"`
-	ExternalLoggerConfiguration string                    `schema:"external_logger_configuration" json:"externalLoggerConfiguration"`
-	DisableJdkLogger            bool                      `schema:"disable_jdk_logger" json:"disableJdkLogger"`
-	MaxCount                    int                       `schema:"max_count" json:"maxCount"`
-	UserExclusionAttributes     []string                  `schema:"user_exclusion_attributes" json:"userExclusionAttributes"`
-	UserMandatoryAttributes     []string                  `schema:"user_mandatory_attributes" json:"userMandatoryAttributes"`
-	AgamaConfiguration          AgamaConfiguration        `schema:"agama_configuration" json:"agamaConfiguration"`
-	AuditLogConf                AuditLogConf              `schema:"audit_log_conf" json:"auditLogConf"`
-	DataFormatConversionConf    DataFormatConversionConf  `schema:"data_format_conversion_conf" json:"dataFormatConversionConf"`
-	Plugins                     []PluginConf              `schema:"plugins" json:"plugins"`
-	AssetMgtConfiguration       AssetMgtConfiguration     `schema:"asset_mgt_configuration" json:"assetMgtConfiguration"`
+	ConfigOauthEnabled               bool                      `schema:"config_oauth_enabled" json:"configOauthEnabled"`
+	DisableLoggerTimer               bool                      `schema:"disable_logger_timer" json:"disableLoggerTimer"`
+	DisableAuditLogger               bool                      `schema:"disable_audit_logger" json:"disableAuditLogger"`
+	CustomAttributeValidationEnabled bool                      `schema:"custom_attribute_validation_enabled" json:"customAttributeValidationEnabled"`
+	ArcValidationEnabled             bool                      `schema:"arc_validation_enabled" json:"arcValidationEnabled"`
```

Comment:
Shouldn't it be `acrValidationEnabled`? https://github.com/JanssenProject/jans/blob/e9e9808857fa58e70e02bd2fc054e7ae15b7200c/jans-linux-setup/jans_setup/templates/jans-config-api/dynamic-conf.json#L7